### ISSUE TITLE
Will the SOLOv2 for panoptic segmentation be published?

### ISSUE THREAD

**Issue by Wei-Mao** [WXinlong/SOLO#106]

**Comment by WXinlong** [comment]
@Wei-Mao  Sorry we currently don't have time to organize the panoptic segmentation part. We may publish those code and models if we are free after the CVPR ddl. But it should be easy to implement yourself, given the [mask feature branch](https://github.com/WXinlong/SOLO/blob/master/mmdet/models/mask_heads/mask_feat_head.py).

**Comment by kemaloksuz** [comment]
I agree it is not difficult to implement the network. But I think it could be better to have an implementation from the authors to be sure about all implementation details.  So, now that the CVPR deadline is over, do you have a plan to release your panoptic segmentation implementation?
